src/ui/CookiesPopup.svelte

```diff
@@ -29,20 +30,33 @@
     isVisible = false
     showManageCookiesDialog()
   }
+
+  function onCloseClick() {
+    isVisible = false
+  }
 </script>
 
 {#if isVisible}
   <div class="cookies border box {className} {style}">
+    {#if $IsMobile}
+      <button class="btn close" on:click={onCloseClick}>
+        <Svg id="close" w="12" />
+      </button>
+    {/if}
```

Comment:
Cookies should be either allowed, rejected or customised. Simple close will break the cookie policy